code/modules/surgery/organs/_organ.dm

```diff
@@ -239,6 +239,12 @@ INITIALIZE_IMMEDIATE(/obj/item/organ)
 		for(var/obj/item/organ/organ as anything in organs)
 			organ.set_organ_damage(0)
 		set_heartattack(FALSE)
+
+		// Ears have aditional vаr "deaf", need to update it too
+		var/obj/item/organ/internal/ears/ears = get_organ_slot(ORGAN_SLOT_EARS)
+		if(ears)
+			ears.adjustEarDamage(0, -120) // full heal ears deafness
```

Comment:
```suggestion
		ears?.adjustEarDamage(0, -120) // full heal ears deafness
```

Reply:
Uhhh, two changes in one code line. Uhhh, github can't digest this review, ok I'm gonna make it by myself